xray/segment.go

```diff
@@ -61,6 +62,13 @@ func BeginSegment(ctx context.Context, name string) (context.Context, *Segment)
 }
 
 func BeginSegmentWithSampling(ctx context.Context, name string, r *http.Request, traceHeader *header.Header) (context.Context, *Segment) {
+	// If SDK is disabled then return with an empty segment
+	disableKey := os.Getenv("AWS_XRAY_SDK_DISABLED")
```

Comment:
Does it make sense to put this in `config.go`? If not, at least we should extract a function for this instead of copying it everywhere.

Reply:
That's a fair point. We can extract a function instead of writing this logic everywhere.